### PR TITLE
Temporary CSS  changes

### PR DESCRIPTION
**Pull request by AnithaDharaneedharan**
Temporary CSS changes

### CODE REVIEW

File: src/frontend/components/Calendar/PickDates.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+export default class PickDates extends Component {
+
+    state = {
+        value: new Date()
+    }
+
+
+    onChange = (value) => {
+        debugger;
```
Comment (Selchuck) [inline src/frontend/components/Calendar/PickDates.js]: "debugger" please remove.

File: src/frontend/components/Calendar/PickDates.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        this.setState({value});
+        alert(`New date is ${value.toString()}`)
+    }
+
+    selectedDate = () => {
+       
+        return `The selected date is ${this.props.value} ` ;
+
```
Comment (Selchuck) [inline src/frontend/components/Calendar/PickDates.js]: indentation required

File: src/frontend/components/Calendar/PickDates.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+                    value= {this.state.value}
+                    onClickDay = {this.selectedDate} 
+                />
+              <div>{users.map(user => user.availability[0])[0].toString()}</div>
+            </div>
+    
+        );
+
+    }
+
```
Comment (Selchuck) [inline src/frontend/components/Calendar/PickDates.js]: indentation required

File: src/frontend/components/Email/Email.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  return (
+    <div>
+      <textarea
+        className="placeholder"
+        placeholder="   Please type your message here!"
+      ></textarea>
+
+      <div className="emailBox">
+    
+
```
Comment (Selchuck) [inline src/frontend/components/Email/Email.js]: indentation required

File: src/frontend/components/Email/Email.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+export default function Email() {
+  return (
+    <div>
+      <textarea
+        className="placeholder"
+        placeholder="   Please type your message here!"
+      ></textarea>
```
Comment (Selchuck) [inline src/frontend/components/Email/Email.js]: indentation required

File: src/frontend/components/Email/Email.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+      <div className="emailBox">
+    
+     
+      </div>
+
```
Comment (Selchuck) [inline src/frontend/components/Email/Email.js]: indentation required

File: src/frontend/components/Results/Results.css [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+.user-image {
+    height: 100px;
+    width: 100px;
+    padding: 20px;
+}
+
+.resultsTable {
+    /* transform: translate(20%, 20%); */
```
Comment (Selchuck) [inline src/frontend/components/Results/Results.css]: comment can be removed

File: src/frontend/components/Results/Results.css [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+    background-color: tomato;
+}
+
+/* .nonDisplayonProfile {
+    display: none;
+} */
```
Comment (Selchuck) [inline src/frontend/components/Results/Results.css]: comments can be removed

File: src/frontend/components/Results/Results.js [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+  constructor() {
+    super();
+    this.state = {filteredUsers: '' , filteredLocation: '' }
+  }
+
+  render() {
+    return (
+      <div>
+      <div>
+        <SearchBar
```
Comment (Selchuck) [inline src/frontend/components/Results/Results.js]: indentation required

File: src/frontend/components/headingFeature.js [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+   
+
+      )
+
+
+
+  
+
+     }
+
```
Comment (Selchuck) [inline src/frontend/components/headingFeature.js]: indentation required

File: src/frontend/components/Results/Results.css (hunk 7 above)
Comment (AnithaDharaneedharan) [reply src/frontend/components/Results/Results.css]: I have left the comments on purpose so that Kainy can use the code later

**Pull request merged**